agent/rpc/peering/service.go

```diff
@@ -394,6 +394,8 @@ func (s *Server) reconcilePeering(peering *pbpeering.Peering) *pbpeering.Peering
 		// reconcile pbpeering.PeeringState_Active
 		if streamState.Connected {
 			cp.State = pbpeering.PeeringState_ACTIVE
```

Comment:
here is where this results in the peering endpoints showing the state as failing